plugins/poll/assets/javascripts/controllers/poll-ui-builder.js.es6

```diff
@@ -20,6 +22,14 @@ export default Controller.extend({
     this._setupPoll();
   },
 
+  @computed("barTypeIdentifier", "pieTypeIdentifier")
```

Comment:
Since `PIE_TYPE` won't ever change, you won't need it to be part of the computed property (which is slower). The dependent keys should only be things that can change.